site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,11 +16,60 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: New manifest field "optional_host_permissions" {: #m102-optional-host-permissions }
+
+April 4, 2022
+
+Added a new manifest key named `optional_host_permissions` for Manifest V3. This behaves the exact
+same as specifying hosts in `optional_permissions` in Manifest V2.
+
+### Chrome 102: injectImmediately property in scripting.executeScript() {: #m102-injectimmediately }
+
+April 4, 2022
+
+`chrome.scripting.executeScript()` now accepts an optional `injectImmediately` property.  If present
+and set to true, the script will inject into the target as soon as possible, rather than waiting for
+`document_idle`. Note that this is not a guarantee the script will inject before the page is loaded
+since the page continues to load while the API call is being made.
+
+### Chrome 102: Omnibox API support in Manifest V3 {: #m102-omnibox }
+
+March 31, 2022
+
+The [Omnibox API](/docs/extensions/reference/omnibox) can now be used in service worker based
+extensions. Previously, some of this API's methods would throw on invocation due to internal
+dependencies on DOM capabilities.
+
+### Chrome 102: wasm-unsafe-eval allowed in Manifest V3 CSP {: #m102-wasm }
+
+March 22, 2022
+
+Manifest V3 extensions can now include `wasm-unsafe-eval` in their `content_security_policy` declarations. This change allows Manifest V3 extensions to use Wasm.
```

Comment:
nit 1: wrap at 80 char
nit 2: maybe avoid "Wasm" since it's not defined?

```suggestion
Manifest V3 extensions can now include `wasm-unsafe-eval` in their `content_security_policy`
declarations. This change allows Manifest V3 extensions to use Web Assembly.
```

Reply:
Ack.

Reply:
Slight tweak to your suggestion: remove the space between "Web" and "Assembly".

```suggestion
Manifest V3 extensions can now include `wasm-unsafe-eval` in their `content_security_policy`
declarations. This change allows Manifest V3 extensions to use WebAssembly.
```